ui/article-aem-renderer/src/main/java/org/cru/godtools/article/aem/activity/AemArticleActivity.kt

```diff
@@ -117,6 +134,24 @@ class AemArticleActivity : BaseSingleToolActivity(false, false) {
         viewModel.article.observe(this, Observer<Article> { onUpdateArticle(it) })
     }
 
+    private fun syncArticle() {
+        val manager = AemArticleManger.getInstance(this)
+        syncTask = when {
+            isValidDeepLink() -> manager.downloadDeeplinkedArticle(articleUri)
+            else -> manager.downloadArticle(articleUri, false)
+        }
+        syncTask.addListener(WeakRunnable(Runnable { onSyncTaskFinished() }), MainThreadExecutor())
+    }
```

Comment:
the `syncArticle` method shouldn't be in this PR, it was combined with some other logic in the firebase PR and renamed to `syncData`